Enabled gzip compression and HTTP/2

marcschwaiger opened the pull request:
Chromes Lighthouse reported poor performance on the ui. 
Compressing data and enabling HTTP/2 were the first tips made by Lighthouse. 
Let's see if this increases performance.
![image](https://user-images.githubusercontent.com/60843765/159891063-38ddee67-becd-460d-b49d-d3b5fc823a2e.png)

marcschwaiger committed:
Enabled gzip compression and HTTP/2

Chromes Lighthouse reported poor performance on the ui. 
Compressing data and enabling HTTP/2 were the first tips made by Lighthouse. 
Let's see if this increases performance.

94726 commented:
Looks good but we settled on using brotli as it is a widely adopted standard and just a better alternative to gzip.
Would be great if you implemented it.

marcschwaiger committed:
Using macbre/nginx-brotli as nginx image

macbre/nginx-brotli has already brotli implemented (also experimental QUIC with HTTP/3)

marcschwaiger committed:
Switched from gzip compression to brotli

marcschwaiger commented:
Linked to [Compression / Optimization #11](https://github.com/LeoTuet/ui-we-want-to-sleep/issues/11)

94726 reviewed (approved):
LGTM, could increase the compression level though

marcschwaiger commented:
Performance score was doubled, warnings now look as following 
![image](https://user-images.githubusercontent.com/60843765/159916196-c1e03d73-d78b-47d0-bbd1-653922dc689f.png)
